### PR TITLE
Add all simple resources from the full.hcl example

### PR DESCRIPTION
**Pull request by MartinPetkov**
This list was generated the following way:
1. Run the engine on the full example.
2. Change all backends to "local".
3. Run init to download all modules into .terraform/ folders.
4. Grep for all unique "resource" declarations and filter out ones in `tfimport.go`.

I will add this script to the CI in a follow-up PR.

WANT_LGTM=all

### CODE REVIEW

File: internal/tfimport/tfimport.go [hunk 1]
```diff
@@ -186,46 +347,57 @@ var importers = map[string]resourceImporter{
...
+	// Helm provider
+	"helm_release": &importer.SimpleImporter{
+		Fields: []string{"namespace", "name"},
+		Tmpl:   "{{.namespace}}/{{.name}}",
+		Defaults: map[string]interface{}{
+			"namespace": "default",
+		},
+	},
+
+	// Kubernetes provider
```
Comment (umairidris) [inline internal/tfimport/tfimport.go:386]: p.s. looks like the k8s provider is getting replaced with https://github.com/hashicorp/terraform-provider-kubernetes-alpha, so you may just want to support that...

File: internal/tfimport/importer/random_integer.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	if err != nil {
+		return "", err
+	}
+	max, err := fromConfigValues("max", rc.Change.After)
+	if err != nil {
+		return "", err
+	}
+
+	// Seed is optional.
+	seed, _ := fromConfigValues("seed", rc.Change.After)
```
Comment (umairidris) [inline internal/tfimport/importer/random_integer.go:46]: seems a bit smelly to ignore errors, maybe return a specific error type and check for that?

Comment (MartinPetkov) [reply internal/tfimport/importer/random_integer.go:46]: I'll replace it with checking if the call returned an error.

File: internal/tfimport/tfimport.go (hunk 1 above)
Comment (MartinPetkov) [reply internal/tfimport/tfimport.go:386]: Where do you see that it's getting replaced?

Comment (MartinPetkov) [reply internal/tfimport/tfimport.go:386]: Since it's regarded as experimental, I'll see if I can support both.

File: cmd/tfimport/main.go [hunk 3]
```diff
@@ -151,7 +151,18 @@ func planAndImport(rn, importRn runner.Runner) (retry bool, err error) {
...
 		if *dryRun {
-			importCmds = append(importCmds, output)
+			cmd := output
+			if err != nil {
+				cmd = err.Error()
+			} else {
+				// The last arg in import could be several space-separated strings. These need to be quoted together.
+				args := strings.Split(cmd, " ")
+				if len(args) >= 4 {
+					cmd = fmt.Sprintf("%v %v %v %q\n", args[0], args[1], args[2], strings.Join(args[3:], ""))
```
Comment (xingao267) [inline cmd/tfimport/main.go:161]: You are joining the args[3:] with just empty string, is that intentional?

Comment (MartinPetkov) [reply cmd/tfimport/main.go:161]: Yes, but looking again I actually meant to use `SplitN`. I'll update that.

**Pull request merged**